courses/task_eight_GP_server/handlers/root.go

```diff
@@ -0,0 +1,123 @@
+package handlers
+
+import (
+    "fmt"
+    "net/http"
+    "log"
+	"io/ioutil"
+	"html/template"
+	"path"
+	"time"
+	"net/url"
+
+    "github.com/Jagrmi-C/gostarted/courses/lib"
+)
+
+type Profile struct {
+	Name    string
+	Address string
+	Token	[]string
+}
+
+func addCookie(w http.ResponseWriter, name string, value string) {
+    expire := time.Now().AddDate(1, 1, 1)
+    cookie := http.Cookie{
+        Name:    name,
+        Value:   value,
+		Expires: expire,
+		MaxAge: 10,
+	}
+	fmt.Println("EX", expire)
+    http.SetCookie(w, &cookie)
+}
+
+func DeleteCookie(w http.ResponseWriter, r *http.Request) {
```

Comment:
Name messing. Reading these names one will think `DeleteCookie` as opposite method for `addCookie`.